modules/code-editor/build.extras.js

```diff
@@ -1,9 +1,15 @@
 module.exports = {
   copyFiles: [
     'src/botpress.d.ts',
-    'src/typings/node.d.ts',
+    'src/typings/node.d.txt',
```

Comment:
why change the extension here?

Reply:
Otherwise typescript uses this file for "global.d.ts" typings since it's closer, and other modules also uses this one